opencomputeproject/SAI
# [SAI-PTF] Add route test cases

## Description of PR
1. Add 15 route test cases.
        sai_route_test.RouteRifTest,
        sai_route_test.RouteRifv6Test,
        sai_route_test.LagMultipleRouteTest,
        sai_route_test.LagMultipleRoutev6Test,
        sai_route_test.DropRouteTest,
        sai_route_test.DropRoutev6Test,
        "sai_route_test.RouteUpdateTest,
        sai_route_test.RouteUpdatev6Test,
        sai_route_test.RouteLPMRouteNexthopTest,
        sai_route_test.RouteLPMRouteNexthopv6Test,
        sai_route_test.RouteLPMRouteRifTest,
        sai_route_test.RouteLPMRouteRifv6Test,
        sai_route_test.SviMacFloodingTest,
        sai_route_test.SviMacFloodingv6Test,
        sai_route_test.SviDirectBroadcastTest
3. Add routev4 and routev6 entry for Device object.
4. Check nbr_entry_v4 status when create_neighbor_by_rif()

## Test
1. Manually run all cases under DUT and they all passed.
2. Run all test on pipeline and they all passed.

Signed-off-by: zitingguo-ms <zitingguo@microsoft.com>
